StyleCop.Analyzers/StyleCop.Analyzers.Test/Helpers/MetadataReferences.cs

```diff
@@ -15,9 +16,9 @@ namespace StyleCop.Analyzers.Test.Helpers
     /// </summary>
     internal static class MetadataReferences
     {
-        internal static readonly MetadataReference CorlibReference = MetadataReference.CreateFromFile(typeof(object).Assembly.Location).WithAliases(ImmutableArray.Create("global", "corlib"));
-        internal static readonly MetadataReference SystemReference = MetadataReference.CreateFromFile(typeof(System.Diagnostics.Debug).Assembly.Location).WithAliases(ImmutableArray.Create("global", "system"));
-        internal static readonly MetadataReference SystemCoreReference = MetadataReference.CreateFromFile(typeof(Enumerable).Assembly.Location);
+        internal static readonly MetadataReference CorlibReference = CreateDotNetFrameworkMetadataReference("mscorlib").WithAliases(ImmutableArray.Create("global", "corlib"));
```

Comment:
Yup, but that one isn't merged yet (and I forgot about it, even though I reviewed it 😉).
I do like the solution here better, as it is more robust towards any future changes.